ffmpeg_plugin/README.md

```diff
@@ -3,56 +3,70 @@
 These instructions build a version of ffmpeg with the SVT-HEVC plugin.  The steps differ depending
 on whether using a Linux or Windows build machine.
 
-For Linux:
-1. Build and install SVT-HEVC 
+## For Linux:
+1\. Build and install SVT-HEVC 
+``` bash
 - git clone https://github.com/OpenVisualCloud/SVT-HEVC
 - cd SVT-HEVC/Build/linux
 - ./build.sh debug|release
 - cd debug|release
 - make install
 - export LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:/usr/local/lib
 - export PKG_CONFIG_PATH=${PKG_CONFIG_PATH}:/usr/local/lib/pkgconfig
```

Comment:
```suggestion
git clone https://github.com/OpenVisualCloud/SVT-HEVC
cd SVT-HEVC/Build/linux
./build.sh release
cd release
make install
export LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:/usr/local/lib
export PKG_CONFIG_PATH=${PKG_CONFIG_PATH}:/usr/local/lib/pkgconfig
```

in the case you are using code blocks, it's best to make the commands as literal as possible